js/data/locale/localematch.json

```diff
@@ -20395,170 +20206,48 @@
         "und-LY": "ar-Arab-LY",
         "und-Lana": "nod-Lana-TH",
         "und-Laoo": "lo-Laoo-LA",
-        "und-Laoo-AU": "hnj-Laoo-AU",
-        "und-Laoo-CN": "hnj-Laoo-CN",
-        "und-Laoo-FR": "hnj-Laoo-FR",
-        "und-Laoo-GF": "hnj-Laoo-GF",
-        "und-Laoo-MM": "hnj-Laoo-MM",
-        "und-Laoo-SR": "hnj-Laoo-SR",
-        "und-Laoo-TH": "hnj-Laoo-TH",
-        "und-Laoo-US": "hnj-Laoo-US",
-        "und-Laoo-VN": "hnj-Laoo-VN",
-        "und-Latn": "en-Latn-US",
-        "und-Latn-419": "es-Latn-419",
-        "und-Latn-AD": "ca-Latn-AD",
+        "und-Latn-AE": "en-Latn-AE",
         "und-Latn-AF": "tk-Latn-AF",
-        "und-Latn-AL": "sq-Latn-AL",
         "und-Latn-AM": "ku-Latn-AM",
-        "und-Latn-AO": "pt-Latn-AO",
-        "und-Latn-AR": "es-Latn-AR",
-        "und-Latn-AS": "sm-Latn-AS",
-        "und-Latn-AT": "de-Latn-AT",
-        "und-Latn-AW": "nl-Latn-AW",
-        "und-Latn-AX": "sv-Latn-AX",
-        "und-Latn-AZ": "az-Latn-AZ",
-        "und-Latn-BA": "bs-Latn-BA",
-        "und-Latn-BE": "nl-Latn-BE",
-        "und-Latn-BF": "fr-Latn-BF",
```

Comment:
Something went wrong with the generation of this file. The mapping

```
"BF": "fr-Latn-BF"
```

exists in the likelyLocalesAdditional.json and the code in genlikelyloc.js is supposed to make sure that it uses that entry to create a new one:

```
"und-Latn-BF": "fr-Latn-BF"
```

But for some reason, all of these are deleted. Can you look at these please?

Reply:
These data were originally in the CLDR cldr-core/supplemental/likelySubtags.json file. not from the likelyLocalesAdditional.json file. and quite many of und-Latn-* are deleted in cldr v46

Reply:
I updated the tool code to reserve the und-Script und-Script-Region case and then regenerated the localematch.json file. Please check them again.

Reply:
Many of these und-<script>-<region> ones are still deleted from this section...

Reply:
yeah.. To avoid losing all und- data, I have added locales to likelyLocalesAdditional.json file then re-generate the localematch.json file. Please check them again.